README.md

```diff
@@ -27,15 +27,16 @@ Below we will provide samples on how to send cloudevents using the popular
 from cloudevents.http import CloudEvent, to_binary
 import requests
 
-
-# This data defines a binary cloudevent
+# "id" attribute defaults to a uniquely generated string
```

Comment:
We still need a summary comment describing what the below lines are doing. Maybe something like:

```py
# Create a CloudEvent
# - The CloudEvent "id" is generated if omitted. "specversion" defaults to "1.0".
```